db/compaction/compaction_picker.h

```diff
@@ -57,7 +57,8 @@ class CompactionPicker {
   virtual Compaction* PickCompaction(const std::string& cf_name,
                                      const MutableCFOptions& mutable_cf_options,
                                      VersionStorageInfo* vstorage,
-                                     LogBuffer* log_buffer) = 0;
+                                     LogBuffer* log_buffer,
+                                     SequenceNumber earliest_memtable_seqno = port::kMaxUint64) = 0;
```

Comment:
s/port::kMaxUint64/kMaxSequenceNumber ?

Reply:
Yes. I didn't know that there was such a variable before.